### ISSUE TITLE
[Bug]: Homebridge restart when disabled the alarm

### ISSUE THREAD

**Issue by JGutierrezAlcala** [homebridge-eufy-security/plugin#94]
### What happened?

When I switch the alarm mode to "Disabled", the homebridge process fail and restart. 

### Device Type

Station (Supported)

### Plugin Version

v2.0.x (Supported)

### HomeBridge Version

other (UnSupported)

### NodeJS Version

v16 (Supported)

### Operating System

Docker (UnSupported)

### Relevant log output

```shell
[6/27/2022, 9:54:23 AM] InvalidPropertyValueError: Value "NaN" isn't a valid value for property "guardMode"
    at validValue (/homebridge/node_modules/homebridge-eufy-security/node_modules/eufy-security-client/src/utils.ts:114:19)
    at Station.setGuardMode (/homebridge/node_modules/homebridge-eufy-security/node_modules/eufy-security-client/src/http/station.ts:484:19)
    at StationAccessory.handleSecuritySystemTargetStateSet (/homebridge/node_modules/homebridge-eufy-security/src/accessories/StationAccessory.ts:212:24)
    at SecuritySystemTargetState.<anonymous> (/homebridge/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Characteristic.ts:1676:42)
    at step (/homebridge/node_modules/homebridge/node_modules/tslib/tslib.js:144:27)
    at Object.next (/homebridge/node_modules/homebridge/node_modules/tslib/tslib.js:125:57)
    at /homebridge/node_modules/homebridge/node_modules/tslib/tslib.js:118:75
    at new Promise (<anonymous>)
    at __awaiter (/homebridge/node_modules/homebridge/node_modules/tslib/tslib.js:114:16)
    at SecuritySystemTargetState.Characteristic.handleSetRequest (/homebridge/node_modules/homebridge/node_modules/hap-nodejs/dist/lib/Characteristic.js:824:38)
[6/27/2022, 9:54:24 AM] Got SIGTERM, shutting down Homebridge...
[6/27/2022, 9:54:24 AM] [EufySecurity-2.0.1] WARN:  Push Closed!
[6/27/2022, 9:54:24 AM] [EufySecurity-2.0.1] INFO:  Finished shutdown!
[6/27/2022, 9:54:29 AM] [HB Supervisor] Homebridge Process Ended. Code: 143, Signal: null
[6/27/2022, 9:54:34 AM] [HB Supervisor] Restarting Homebridge...
```

**Comment by thieren** [comment]
Hi @JGutierrezAlcala 

I believe this to be a duplicate of #52 

The issue is already fixed in the latest beta and will be in future releases.

Thx for your feedback!

**Comment by JGutierrezAlcala** [comment]
No need to wait for the new release, this user has found workaround for the bug:

https://github.com/homebridge-eufy-security/plugin/issues/52#issuecomment-1146026568

It works for me. Thanks!

**Comment by thieren** [comment]
Is fixed with release of 2.1.0
